src/parser/jobs/blm/modules/Gauge.tsx

```diff
@@ -549,15 +558,15 @@ export class Gauge extends CoreGauge {
 			}))
 		}
 
-		if (this.overwrittenPolyglot > 0) {
+		if (this.overwritePolyglotTimestamps.length > 0) {
```

Comment:
wrt the other comment about holding a single array containing all of the data, it'll be better to do something like
const overwrittenPolyglot = this.overwriteData.filter(event => etc.).length
and use that const in the condition and the Plural rather that calculating the count twice